### PR TITLE
Fix(docs): Fixing a bunch of small typos

### PR DESCRIPTION
**Pull request by rettinghaus**

### CODE REVIEW

File: source/docs/09-textencoding.xml [hunk 1]
```diff
@@ -90,7 +90,7 @@
...
+            <p>This chapter describes methods for encoding textual content with MEI. It is divided into section: One part deals with <ptr target="#textstructures"/> in MEI, the other with <ptr target="#text"/>. While the first covers structures of textual documents such as front- and back matter, the latter describes how to mark up features and various entities within a text, such as names, tables or quotes. These features may appear both within data (a <gi scheme="MEI">fig</gi>ure scribbled as marginal <gi scheme="MEI">annot</gi>ation into a score by a bored second violin…) and metadata (a number of <gi scheme="MEI">date</gi>s within a text about the <gi scheme="MEI">creation</gi> of a work). Accordingly, many of the elements and models explained in this chapter are used to encode <ptr target="#metadata"/>. However, they should not be confused with the elements from the <ptr target="#lyricsperfdir"/> chapter, which deals with <hi rend="italic">performed</hi> text in MEI.</p>
```
Comment (musicEnfanthen) [inline source/docs/09-textencoding.xml]:
```suggestion
            <p>This chapter describes methods for encoding textual content with MEI. It is divided into section: One part deals with <ptr target="#textstructures"/> in MEI, the other with <ptr target="#text"/>. While the first covers structures of textual documents such as front matter or back matter, the latter describes how to mark up features and various entities within a text, such as names, tables or quotes. These features may appear both within data (a <gi scheme="MEI">fig</gi>ure scribbled as marginal <gi scheme="MEI">annot</gi>ation into a score by a bored second violin…) and metadata (a number of <gi scheme="MEI">date</gi>s within a text about the <gi scheme="MEI">creation</gi> of a work). Accordingly, many of the elements and models explained in this chapter are used to encode <ptr target="#metadata"/>. However, they should not be confused with the elements from the <ptr target="#lyricsperfdir"/> chapter, which deals with <hi rend="italic">performed</hi> text in MEI.</p>
```

**Review by musicEnfanthen — COMMENTED** [review]
Good catches. Thank you @rettinghaus. One small addition below.

**Review by musicEnfanthen — DISMISSED** [review]

File: source/docs/01-introduction.xml [hunk 2]
```diff
@@ -604,7 +604,7 @@
                   </list>
                   <p>The first three profiles provide good starting points to encode music from the
                      respective repertoires. They may also serve as template for further,
-                     project-specific customizations. The latter two profiles are targetting very
+                     project-specific customizations. The latter two profiles are targeting very
```
Comment (ahankinson) [inline source/docs/01-introduction.xml]:
```suggestion
                     project-specific customizations. The latter two profiles target very
```

**Review by ahankinson — DISMISSED** [review]
Just a small wordsmithing change as a suggestion; otherwise looks good.

**Review by musicEnfanthen — APPROVED** [review]

**Review by ahankinson — APPROVED** [review]

**Comment by ahankinson** [comment]
Thanks!

**Pull request merged**